init-plugin.sh

```diff
@@ -16,15 +16,16 @@ if [ -z "$name" ]; then
 	exit 1
 fi
 
-if ! perl -pe '/^[A-Z][a-z0-9]*( [A-Z][a-z0-9]*)*$/ || exit 1;' > /dev/null <<< "$name"; then
-	echo "Malformed name argument '$name'. Please use title case words separated by spaces. No hypens."
+valid="^[A-Z][a-z0-9]*( [A-Z][a-z0-9]*)*$"
+if [[ ! "$name" =~ $valid ]]; then
+	echo "Malformed name argument '$name'. Please use title case words separated by spaces. No hyphens."
 	exit 1
 fi
 
-slug=$( perl -pe '$_ = lc; s/ /-/g' <<< "$name" )
-prefix=$( perl -pe '$_ = lc; s/ /_/g' <<< "$name" )
-namespace=$( perl -pe 's/ //g' <<< "$name" )
-class=$( perl -pe 's/ /_/g' <<< "$name" )
+slug="$( echo "$name" | tr '[:upper:]' '[:lower:]' | sed 's/ /-/' )"
+prefix="$( echo "$name" | tr '[:upper:]' '[:lower:]' | sed 's/ /_/' )"
+namespace="$( echo "$name" | sed 's/ //' )"
+class="$( echo "$name" | sed 's/ /_/' )"
```

Comment:
The above `sed` calls are missing the `g` modifier for global replacements, I believe.